src/styles/blocks/_contact.scss

```diff
@@ -0,0 +1,96 @@
+.contact {
+  &__wrapper {
+    @include pageGrid;
+    justify-content: space-between;
+    row-gap: 48px;
+  }
+
+  &__form {
+
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-column: span 5;
+    }
+
+  }
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-column: 8 / 11;
+    }
+  }
+
+  &__input {
+    box-sizing: border-box;
```

Comment:
Here and in other places - it is better to set box-sizing globally via `*` (if you don't need to have different values on different components)